contrib/sarplus/python/tests/sample-input.txt

```diff
@@ -0,0 +1,8 @@
+i1,i2,value
```

Comment:
Do we need this?

Reply:
using the file as part of the unit tests. referring to the headers?